javascript/javascript3/week2/converter.js

```diff
@@ -0,0 +1,87 @@
+const inputEl = document.getElementById("inputEl");
+const convertFrom = document.getElementById("convertFrom");
+const convertTo = document.getElementById("convertTo");
+const convertBtn = document.getElementById("btn");
+const api = `https://open.er-api.com/v6/latest/`;
```

Comment:
Should all variables be global, or some can be local?

Reply:
The only one that can be local is the inputEl, because it was used only once. (fixed).
The others can not because were used inside different functions.